src/graphql/queries.graphql

```diff
@@ -44,13 +39,8 @@ query QuestChainSearch($search: String, $limit: Int) {
   }
 }
 
-query QuestChainSearch($search: String, $limit: Int) {
```

Comment:
Also why are you removing this search option? 
We need this for the "search" functionality to work. 
We should rather create a new query instead of replacing this.

I'm also thinking we need a query for fetching ALL slugs so that when a user is creating a new slug we can check and make sure there are no conflicts.

But I'm still worried there might be a race condition here where two people set the slug at the same time, and there is no way for us to stop that from happening.

Reply:
> Also this might be a mistake?

there were 2 exactly queries, which were exactly the same, so i replaced one of them.

> I'm also thinking we need a query for fetching ALL slugs so that when a user is creating a new slug we can check and make sure there are no conflicts.

we'd need to fetch all quest chains and return only their slugs for this to work, right?

> race condition

also let's not think about this edge case